flake.nix

```diff
@@ -324,6 +324,8 @@
               buildkite-test-collector-rust
               docker-compose
               jq
+              nats-top
+              natscli
```

Comment:
Should be able to test more directly now!

Reply:
I feel like I've been hoarding the NATS CLI to myself--no longer!